src/test/suite/language/mongoDBService.test.ts

```diff
@@ -2441,22 +2960,31 @@ suite('MongoDBService Test Suite', () => {
       expect(result).to.deep.equal(expectedResult);
     });
 
-    test('includes results from print() and console.log()', async () => {
+    test('sends print() and console.log() output continuously', async () => {
       const source = new CancellationTokenSource();
+
+      const consoleOutputs: unknown[] = [];
+
+      Sinon.stub(connection, 'sendNotification')
+        .withArgs(ServerCommands.SHOW_CONSOLE_OUTPUT)
+        .callsFake((_, params) =>
+          Promise.resolve(void consoleOutputs.push(...params))
+        );
+
       const result = await testMongoDBService.evaluate(
         {
           connectionId: 'pineapple',
           codeToEvaluate: 'print("Hello"); console.log(1,2,3); 42',
         },
         source.token
       );
+
+      Sinon.restore();
```

Comment:
With `Sinon.restore()` we usually call it in `afterEach` functions to ensure that even if the test fails we restore any stubs. That way if it does fail we don't impact other tests.